### DISCUSSION TITLE
Wrapping text in a translation row behaviour

### DISCUSSION THREAD

**Discussion by richardcave27** [BigSoftVideo/DOTE#7] [General]
![image](https://user-images.githubusercontent.com/42712514/209045039-d737a720-be24-498e-92bc-79f2151faebd.png)

I am attaching some lines from a transcription I am writing now. I added a translation row (Pam.Rel) and am using it to reflect automated captioning that is going in parallel with the talk (Pam). This works well, except when I want to wrap the translation row into two  - to make the row not too wide. 

Example 1 shows how I have wrapped the talk row (Pam:) into two rows, but the translation row (Pam.Rel) is left on one row. Pam.Rel is too long for ease of visibility. Note I have overlapping talk which I have reflected in row 47 and row 49. This is fine and DOTE is not showing any errors.

If I then wrap Pam.Rel as well as Pam: to make the script more readable, I get errors which I cannot fix unless I revert Pam.Rel to one line
![image](https://user-images.githubusercontent.com/42712514/209045793-7efb9ea7-d908-44a8-91f6-6f57aabf9cec.png)

I can fix this when I export the file to a word doc or somewhere, but it would be better if I could just do in in Dote. Would you have any advice here?

Thanks so much
Richard

**Comment by skandilocks** [comment]
_DOTE_ always expects the following basic structure in a Jeffersonian neighbourhood:
- Primary speaker (one line)
- Translation subtier (one line)
- Secondary speaker in overlap (one line)
- Translation subtier (one line)

For more details about the structure of a Jeffersonian neighbourhood: https://bigsoftvideo.github.io/DOTE/jefferson.html

Your transcript has broken that pattern by continuing the Primary speaker line onto the next line (as if word wrapping). We don't support this for a number of reasons, including parseability, style and conformity to Jeffersonian conventions. For example, how is _DOTE_ to figure out if line 47 or 49 is a continuation or a next speaker?

The solution could be to do what _DOTE_ would prefer:

![image](https://user-images.githubusercontent.com/34093890/209099240-870a5e34-1369-46a0-90b6-2fc1180a3f43.png)

BTW, why is the captioning so different from the main speaker lines? Is this the result of bad AI speech recognition?

**Comment by richardcave27** [comment]
Hi Paul, 

Thank you for such a quick reply – really helpful!

My PhD is on the Google Relate app https://blog.google/outreach-initiatives/accessibility/project-relate/ and how it may support people with significant dysarthria to be better understood. Pam has severe dysarthria and has ‘trained’ the Relate app, but only to the minimum required. The Relate model is showing a high word error rate, though in other parts of the conversation appears to help the listener - even though it is not accurate. 

Really happy to talk about the Relate project further if you are interested.

Thanks again

Richard
